core/state/access_events_test.go

```diff
@@ -41,8 +41,9 @@ func TestAccountHeaderGas(t *testing.T) {
 
 	// Check cold read cost
 	gas := ae.VersionGas(testAddr, false)
-	if gas != params.WitnessBranchReadCost+params.WitnessChunkReadCost {
-		t.Fatalf("incorrect gas computed, got %d, want %d", gas, params.WitnessBranchReadCost+params.WitnessChunkReadCost)
+	want := params.WitnessBranchReadCost+params.WitnessChunkReadCost
+	if gas != want {
```

Comment:
```suggestion
	if want := params.WitnessBranchReadCost+params.WitnessChunkReadCost; gas != want {
```
And the same for all the cases further down